pkg/network/tracer/connection/kprobe/config.go

```diff
@@ -9,13 +9,30 @@ package kprobe
 
 import (
 	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
```

Comment:
This should be grouped into the imports below, see https://github.com/DataDog/datadog-agent/blob/main/docs/dev/imports.md